ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/monitor-layer1.j2

```diff
@@ -0,0 +1,32 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - monitor layer1 #}
+{% if monitor_layer1.logging_mac_fault is arista.avd.defined(true) or monitor_layer1.logging_transceiver is arista.avd.defined %}
+
+## Monitor Layer1 Logging
+
+| Logging |
+| ------- |
+{%     if monitor_layer1.logging_mac_fault is arista.avd.defined(true) %}
+| mac fault |
+{%     endif %}
+{%     if monitor_layer1.logging_transceiver.enable is arista.avd.defined(true) %}
+{%         if monitor_layer1.logging_transceiver.dom is arista.avd.defined(true) %}
+| transceiver dom |
+{%         endif %}
+{%         if monitor_layer1.logging_transceiver.communication is arista.avd.defined(true) %}
+| transceiver communication |
+{%         elif monitor_layer1.logging_transceiver.dom is not arista.avd.defined %}
+| transceiver |
```

Comment:
Are we sure about this logic here?

```
host3(config-mon-layer1)#show active
monitor layer1
   logging transceiver communication
   logging mac fault
```

Reply:
Tested on 7280CR3, it seems that the command `logging transceiver` turns on both dom and communication. I propose we only support explicitly setting the `dom` and `communication` options rather then relying on the `logging transceiver` to enable both (or whatever the platform supports).

Reply:
I observed the same and I agree :) I would remove the `enable` and keep only `dom` and `communication`